src/utils/src/deviceinfo.cpp

```diff
@@ -26,6 +26,24 @@ bool DeviceInfo::hasRoundScreen()
 #endif
 }
 
+double DeviceInfo::edgeBoundary()
+{
+#ifdef EDGE_BOUNDARY
+    return EDGE_BOUNDARY;
+#else
+    return 0.1;
+#endif
+}
+
+int DeviceInfo::hasFlatTire()
```

Comment:
Given that this returns the actual  size of the flat tire, I believe that flatTireHeight would be a better name than hasFlatTire